src/components/RoutePage.vue

```diff
@@ -10,6 +10,9 @@
       <i class="material-icons">done</i>
     </button>
   </section>
+  <section v-else>
+    <h1>Congratulations! You have completed all the pubs 🍻</h1>

```

Comment:
Nit: "You have completed all the pubs" reads a little bit weirdly to me. Maybe something like "You have completed the pub crawl!"? Not urgent.

Reply:
Agreed 👍